docs/customer_management/cp_configuration.md

```diff
@@ -0,0 +1,100 @@
+---
+description: Configure Customer Portal to fit the needs of your business.
+edition: experience
+---
+
+# Customer Portal configuration
+
+You can overwrite the default configuration of the Customer Portal
+to fit its capabilities to the unique needs of your business.
+
+## `corporate` SiteAccess
+
+The predefined `corporate` SiteAccess in `corporate_group`
+(configured in `config/packages/ibexa.yaml`) serves the Customer Portal.
+If you need a multisite setup with multiple Customer Portals,
+add any additional SiteAccesses to `corporate_group`.
+
+## Customer identifier
+
+`ibexa_default_settings.yaml` contains setting that indicate which Content Types are treated like customers:
```

Comment:
This is not true, that settings just say what content types should be treated like `Users` in terms for example usage in `UserService`.
There is no way to assign which CT should be treated like `cutomer`, as there is no specific behavior for that.